### DISCUSSION TITLE
[TOOL]Eluna Simplifier (Atom - Package)

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#3967] [General]
I come to share with you a Package that I am currently developing.

It allows you to develop with Eluna faster, without going through the Wiki to get the Methods or RegisterEvents.

For the moment there are only Player RegisterEvent and Methods.

But this will be updated as we go along.

https://atom.io/packages/eluna-simplifier

_If you are looking for a RegisterPlayerEvent_

1. ![1](https://user-images.githubusercontent.com/58150497/69533392-086bdc00-0f78-11ea-9a3d-27b278397deb.png)
2. ![2](https://user-images.githubusercontent.com/58150497/69533393-086bdc00-0f78-11ea-9130-b784298b5ed8.png)

_If you are looking for a Player Method_
1. ![3](https://user-images.githubusercontent.com/58150497/69533395-086bdc00-0f78-11ea-886b-6da08fab304b.png)
2. ![4](https://user-images.githubusercontent.com/58150497/69533397-09047280-0f78-11ea-9e5d-ff61da3f3047.png)

**Comment by BarbzYHOOL** [comment]
Nice

**Comment by ghost** [comment]
If you want to install the package, you can now do so.

It is available at this address: https://atom.io/packages/eluna-simplifier

Once installed, restart Atom and load your lua file.
Press ctrl + shift + L and choose Eluna.

Once this is done, you have access to all Eluna Simplifier "shortcuts".
To View all of RegisterEvent list simply type: 

Displays

A proposal should appear you just have to press enter

**Comment by FrancescoBorzi** [comment]
Starred!
